Classes/common/CDTIncrementalStore.m

```diff
@@ -16,6 +16,14 @@
 #import "CDTFieldIndexer.h"
 #import "CDTISGraphviz.h"
 
+@interface CDTISBatchUpdateResult : NSPersistentStoreResult
+@property (nonatomic, strong) id result;
+@property (nonatomic, assign) NSBatchUpdateRequestResultType resultType;
```

Comment:
This looks promising but sadly fails at runtime with:

    "NSInvalidArgumentException", "-[NSBatchUpdateResult setResultType:]: unrecognized selector sent to instance 0x7fb273307260"